File: Array/1450.Number of Students Doing Homework at a Given Time/Solution.java
class Solution {
    public int busyStudent(int[] startTime, int[] endTime, int queryTime) {
        int[] q = new int[1010];
        for (int v : startTime) {
            q[v]++;
        }
        for (int v : endTime) {
            q[v + 1]--;
        }
        return Arrays.stream(q).limit(queryTime + 1).sum();
    }
}

File: Array/1450.Number of Students Doing Homework at a Given Time/Solution.py
class Solution:
    def busyStudent(self, startTime: List[int], endTime: List[int], queryTime: int) -> int:
        t = [0] * 1010
        for v in startTime:
            t[v] += 1
        for v in endTime:
            t[v + 1] -= 1
        return sum(t[:queryTime + 1])